### DISCUSSION TITLE
install_routes_xfrmi how to use

### DISCUSSION THREAD

**Discussion by tiantian201202** [strongswan/strongswan#2173] [General]
How to use install_routes xfrmi？

**Comment by tobiasbrunner** [comment]
Try reading [the docs](https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html#_xfrm_interfaces_on_linux).

**Comment by tiantian201202** [comment]
Can you provide a configuration example

**Comment by tiantian201202** [comment]
like this?in strongswan.conf
charon {
  plugins {
    kernel-netlink {
      install_routes_xfrmi = yes
    }
  }
}

**Reply by celesteking** [reply]
That usually resides in `strongswan.d/charon/kernel-netlink.conf`, but yes, that's correct.
